R/standard_who.R

```diff
@@ -222,7 +221,7 @@ who_value2centile <- function(
 #' @export
 #' @rdname who_var2zscore
 #' @examples
-#' haz <- who_htcm2zscore(cpp$agedays, cpp$htcm, cpp$sex)
+#' cpp$haz <- who_value2zscore(x = agedays,  y = lencm, sex = sex, data = cpp)
```

Comment:
Why did this example change?
```suggestion
#' haz <- who_htcm2zscore(cpp$agedays, cpp$htcm, cpp$sex)
```

Reply:
There were places in the help files where you were referencing the `haz` that was previously in `cpp`.  Your fix looks correct here.